lighthouse-core/test/audits/__snapshots__/metrics-test.js.snap

```diff
@@ -3,7 +3,7 @@
 exports[`Performance: metrics evaluates valid input (with lcp from all frames) correctly 1`] = `
 Object {
   "cumulativeLayoutShift": 0.0011656245471340055,
-  "cumulativeLayoutShiftAllFrames": 0.4591700003057729,
+  "cumulativeLayoutShiftAllFrames": undefined,
```

Comment:
This snapshot is using the old trace without the weighted score, so CLS-AF is undefined.